en/admin/installation/virtualhosts.md

```diff
@@ -33,6 +33,30 @@ Installation in folders can work, but is not supported by the maintainers.
         </IfModule>
     </Directory>
 
+    <IfModule mod_expires.c>
```

Comment:
Maybe you can add a comment about commenting these following lines if people don't want a caching strategy on their assets

Reply:
Done.